learn/configuration/instance_options.md

```diff
@@ -125,7 +125,7 @@ Meilisearch automatically collects data from all instances that do not opt out u
 
 **Environment variable**: `MEILI_DUMPS_DIR`
 **CLI option**: `--dumps-dir`
-**Default value**: `dumps/`
+**Default value**: `dumps/`/`"meili_data/dumps"` (For Docker)
```

Comment:
Same, the default is `dumps`
For docker it's `meili_data/dumps` because the working directory is `meili_data` if I'm correct
Same, I ping you @irevoire 

Plus it's not "for docker" but "for the official meilisearch docker image". If you use Docker, the default path and dump path are not in `meili_data`

That's why I'm not sure we want to specify this

Reply:
![image](https://user-images.githubusercontent.com/17902415/170092078-0f239b85-67a0-4eea-babc-31177b765bbb.png)